app/controllers/system_admin/accredited_providers/confirmations_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module SystemAdmin
+  module AccreditedProviders
+    class ConfirmationsController < ApplicationController
+      # TODO: Verify that auth checks are happening correctly
+
+      before_action :enforce_feature_flag
+
+      def show
+        trainee = Trainee.find(params[:trainee_id])
```

Comment:
I think we may want to reference the trainee in the views for back links etc so would be good to memoise the call. I've done it here in the follow up PR https://github.com/DFE-Digital/register-trainee-teachers/pull/3616/files#diff-7b157ff817a403a02eb69d83ebfeb2c041a454a77438c9c16c2a6d28683cfc43R31